### DISCUSSION TITLE
January 15 2025 Meetup - Boston MA USA

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#13165] [General]
I'll be organizing a Boston MA DataFusion meetup on Wed January 15 2025

Looking for speakers and people to help organize 

It will very likely be at the Boston Starburst offices 
Thanks to Starburst 🙏  for the space,

**Comment by matthewmturner** [comment]
I have family in the area so could potentially come to this.

**Comment by Omega359** [comment]
I am definitely interested in this one. Well, I'm interested in them all but I may actually be able to attend this one :)

**Comment by alamb** [comment]
Here is the link for registration: https://lu.ma/uwis1x7t

**Comment by alamb** [comment]
Getting excited -- 1 week away -- we have about 45 signups so far

**Comment by matthewmturner** [comment]
Wish I could make this but im traveling on the 18th so it's just a bit too much travel in a short period of time for me - I look forward to seeing the presentations after though. ive been going through the chicago ones and they are great.

**Comment by alamb** [comment]
Here are some photos I took at the even:

![PXL_20250115_212239922](https://github.com/user-attachments/assets/4117acb7-2dab-48ea-a407-f69ecc508382)
![PXL_20250115_221126783](https://github.com/user-attachments/assets/e80bbc73-9213-4c85-90fb-baf39c7b0004)
![PXL_20250115_223656993](https://github.com/user-attachments/assets/4d65652f-99b3-43c3-9872-521ed651e718)
![PXL_20250115_223658241](https://github.com/user-attachments/assets/3821db5c-722b-4449-b311-748eb2fdf2fd)
![PXL_20250115_223708028](https://github.com/user-attachments/assets/769abda3-5726-4636-b7a3-426f6eb23cd1)
![PXL_20250115_232224418](https://github.com/user-attachments/assets/b019d88d-ac57-4272-9098-3cf5336793fe)
![PXL_20250115_232228308](https://github.com/user-attachments/assets/3f9653e1-d4bd-4c87-b318-7eb080192107)
![PXL_20250115_232233087](https://github.com/user-attachments/assets/8648eb58-5207-48ed-a9ee-44ce87680f94)
![PXL_20250115_232234900](https://github.com/user-attachments/assets/bd786939-6f01-42f7-801f-89fb5763d56f)

**Comment by alamb** [comment]
It was a good time -- I messed up the AV recording so the talks were not recorded this time I don't think
